tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest3/testDummyStep_MC/vuq_param.txt

```diff
@@ -0,0 +1,2 @@
+k_rhol = $RAVEN-rhoL$ ! Liquid density from report NUREG-CR-3373
+k_rhov = $RAVEN-rhoV$   ! Vapor density calculated from ideal gas law and assuming 1.225 kg/m^3 at STP
```

Comment:
Seems to me, you do not need this file. This file will generated by raven, right?